File: Refrigtz/FormKindOfGameContinue.vb
Imports System.Windows.Forms
Imports System.Text
Imports System.Linq
Imports System.Drawing
Imports System.Data
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System

Namespace Refrigtz
	Public Partial Class FormKindOfGameContinue
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub FormKindOfGameContinue_Load(sender As Object, e As EventArgs)
			Dim iii As Integer = 0
			Dim t As Integer = DateTime.Now.Second * 1000 + DateTime.Now.Millisecond
			Do
				If DateTime.Now.Second * 1000 + DateTime.Now.Millisecond - t > 1000 Then
					Exit Do
				End If
				iii += 1
				If System.IO.File.Exists("Database\Games\CurrentBank" + iii.ToString() + ".accdb") Then
					comboBoxDataBase.Items.Add("CurrentBank" + iii.ToString() + ".accdb")
				End If
			Loop While True

		End Sub
	End Class

End Namespace

File: Refrigtz/FormKindOfGameContinue.cs
using System;
using System.Windows.Forms;
namespace Refrigtz
{
    [Serializable]
    public partial class FormKindOfGameContinue : Form
    {
        public FormKindOfGameContinue()
        {
            InitializeComponent();
        }

        private void FormKindOfGameContinue_Load(object sender, EventArgs e)
        {
            int iii = 0;
            int t = DateTime.Now.Second * 1000 + DateTime.Now.Millisecond;
            do
            {

                if (DateTime.Now.Second * 1000 + DateTime.Now.Millisecond - t > 1000) break; iii++; if (System.IO.File.Exists(FormRefrigtz.Root + "\\Database\\Games\\CurrentBank" + iii.ToString() + ".accdb")) { ComboBoxDatabase.Items.Add("CurrentBank" + iii.ToString() + ".accdb"); }
            } while (true);

        }

        private void RadioButtonComputerByComputer_CheckedChanged(object sender, EventArgs e)
        {

        }

        private void ComboBoxDatabase_SelectedIndexChanged(object sender, EventArgs e)
        {

        }
    }
}
